### ISSUE TITLE
[Keyboard Navigation – Make Code - Control Pane] Keyboard focus does not move to newly opened blade.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3669]
User Experience:
If focus order is not sequential and meaningful, then keyboard user will find hard to understand the actual flow of page and the actual feature of the page which degrades the user experience.

Note: User credentials should NOT be included in the bug.

Repro Steps:

Launch application URL
Home page will open.
Navigate "New Project" option and select it.
Editor screen will open.
Navigate to "Loops, Logic, Variable, Math" controls and select it.
Navigate through all the controls appearing after selection of element.
Verify the issue.

Actual Result:
Keyboard focus does not move to newly opened blade.

Expected Result:
Keyboard focus should be move to first interactive element of the newly opened blade.

Note:
This issue also exists with Firefox and Chrome Browser.
This issue exists throughout the application

WCAG Reference:
[WCAG 3.2.1 - On Focus (06)](https://www.w3.org/WAI/WCAG21/Understanding/on-focus.html)

Test Environments:
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG3.2.1, #Win10-Edge(Chromium)

[Uploading MAS3.2.1_Focus does not moves to newly open blade.zip…]()
